src/main/kotlin/org/rust/lang/core/psi/ext/RsAssocTypeBinding.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.psi.ext
+
+import com.intellij.lang.ASTNode
+import com.intellij.psi.PsiElement
+import com.intellij.psi.stubs.IStubElementType
+import org.rust.lang.core.psi.RsAssocTypeBinding
+import org.rust.lang.core.psi.RsPath
+import org.rust.lang.core.resolve.ref.RsAssocTypeBindingReferenceImpl
+import org.rust.lang.core.resolve.ref.RsReference
+import org.rust.lang.core.stubs.RsAssocTypeBindingStub
+
+val RsAssocTypeBinding.parentPath: RsPath
+    get() = ancestorStrict()!! // non-null guaranteed by the grammar
```

Comment:
Unfortunately, it's not true
Current grammar allows writing following things
```rust
fn main() {
    let a = S(123);
    let b = a.foo::<Ty = i32>();
}
```
It throws NPE becase `a.foo::<Ty = i32>()` is `RsMethodCall`